app/pages/BeelieversAuction/Info.tsx

```diff
@@ -177,6 +178,50 @@ function InstructionDetails() {
 	);
 }
 
+const EligibleStatusBadge = ({ type }: { type?: AuctionAccountType }) => {
+	if (!type || type === AuctionAccountType.DEFAULT) return null;
+
+	const getStatusConfig = () => {
+		switch (type) {
+			case AuctionAccountType.TESTNET_WHITELIST:
+				return {
+					text: "Congrats! You have Testnet WL tier",
+					gradient: "from-blue-500/20 to-cyan-500/20",
+					border: "border-blue-400/40",
+					textColor: "text-blue-300",
+					outerGradient: "from-blue-500/10 via-cyan-500/5 to-transparent",
+				};
+			case AuctionAccountType.PARTNER_WHITELIST:
+				return {
+					text: "Congrats! You are verified Partner WL",
+					gradient: "from-purple-500/20 to-pink-500/20",
+					border: "border-purple-400/40",
+					textColor: "text-purple-300",
+					outerGradient: "from-purple-500/10 via-pink-500/5 to-transparent",
+				};
+			default:
+				return null;
+		}
+	};
```

Comment:
this function should be outside of the component

Reply:
as discussed merging this